doc/rados/operations/pools.rst

```diff
@@ -94,19 +94,11 @@ To get even more information, you can execute this command with the ``--format``
 Creating a Pool
 ===============
 
-Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_.  Your
-Ceph configuration file contains a setting (namely, ``pg_num``) that determines
-the number of PGs.  However, this setting's default value is NOT appropriate
-for most systems.  In most cases, you should override this default value when
-creating your pool.  For details on PG numbers, see `setting the number of
-placement groups`_
-
-For example:
-
-.. prompt:: bash $
-
-    osd_pool_default_pg_num = 128
-    osd_pool_default_pgp_num = 128
+Before creating a pool, consult `Pool, PG and CRUSH Config Reference`_. The
+Ceph central configuration database in the monitor cluster contains a setting
+(namely, ``pg_num``) that determines the number of PGs per pool. It is possible
```

Comment:
As written this sounds like pg_num applied to all pools.  It is however an attribute of *each* pool maintained I think in the OSD map.  I think you’re talking about 
Osd_pool_default_pg_num

Reply:
\<ed macmahon\>You are correct, sir\.<\/ed macmahon\>

This was last updated in 2013, when, for all I know, there may well have been a variable called "pg_num".

In short: Accepted.

Reply:
As of Dumpling in 2013, the state of the Cephalopod in this respect was as it is now.  ISTR though that the default value of `osd_pool_default_pg_num` at the time was 8.  That was a time when our 450 OSD clusters were pushing the envelope.